requirements.txt

```diff
@@ -1,120 +1,120 @@
-affine==2.4.0
-aiobotocore==2.12.1
-aiohttp==3.9.3
-aioitertools==0.11.0
-aiosignal==1.3.1
-alembic==1.13.1
-annotated-types==0.6.0
-appdirs==1.4.4
-asciitree==0.3.3
-attrs==23.2.0
-blosc2==2.5.1
-bokeh==3.3.4
-botocore==1.34.51
-Brotli==1.1.0
-cached-property==1.5.2
+# affine==2.4.0
```

Comment:
could you remove them, not comment them out. They will be in the git history

Reply:
I have removed extra requirements now